tests/test_output/module_dict.out

```diff
@@ -4,10 +4,8 @@ Layer (type:depth-idx)                   Output Shape              Param #
 ModuleDictModel                          [1, 10, 1, 1]             --
 ├─ModuleDict: 1-1                        --                        --
 │    └─Conv2d: 2-1                       [1, 10, 1, 1]             910
-│    └─MaxPool2d: 2-2                    --                        --
-├─ModuleDict: 1-2                        --                        --
-│    └─LeakyReLU: 2-3                    [1, 10, 1, 1]             --
-│    └─PReLU: 2-4                        --                        1
```

Comment:
We lost a couple layers here

Reply:
Yeah, this is one side-effect/feature of making it based only on forward call. Since maxpool2d and PRelu was not called inside `forward`, they are not included in summary_list and summary.